start-goss-servers.sh

```diff
@@ -35,7 +35,7 @@ fi
 export KUBECONFIG=/etc/kubernetes/admin.conf
 
 # necessary for test that need to know the current hostname
-HOSTNAME=$(hostname -s | grep -Eo '(ncn-[msw][0-9]{3}|.*-pit)$')
+HOSTNAME=$(hostname -s | grep -Eo '(ncn-[msw][0-9]{3}|.*pit)$')
```

Comment:
Maybe just replace `grep` with `sed` - so that if there's unexpected hostname, it remains unchanged?

Reply:
Okay, that will need retesting.   I'll circle back with this suggestion next week.

Reply:
Makes sense. Thank you for returning to this topic!